org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/XMLSyntaxDiagnosticsTest.java

```diff
@@ -217,7 +278,7 @@ public void testETagRequired() throws Exception {
 				"			</Nm>  ";
 		Diagnostic d = d(1, 5, 1, 7, XMLSyntaxErrorCode.ETagRequired);
 		testDiagnosticsFor(xml, d);
-		testCodeActionsFor(xml, d, ca(d, te(1, 8, 1, 8, "</Nm>")));
+		testCodeActionsFor(xml, d, ca(d, te(2, 2, 2, 2, "</Nm>")));

```

Comment:
From https://github.com/eclipse/lemminx/pull/760#discussion_r435431827, it seems like "adding" whitespace to the end of the text node is ok, so what about "removing" whitespace at the end of the textnode?

The behaviour defined in this test is:
```
<UltmtDbtr>
	<Nm>Name
	</UltmtDbtr>
		</Nm>
```

to

```
<UltmtDbtr>
	<Nm>Name
	</Nm></UltmtDbtr>
		</Nm>
```

But would it be better if we "remove" the whitespace and newline from `Name\n____` to have
```
<UltmtDbtr>
	<Nm>Name</Nm>
	</UltmtDbtr>
		</Nm>
```
instead? Ie, insert the codeaction content right after `Name` instead of `Name\n____`?

Reply:
I agree, fixed